ee/tables/homebrew/upgradeable.go

```diff
@@ -0,0 +1,146 @@
+//go:build !windows
+// +build !windows
+
+package brew_upgradeable
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log/slog"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/kolide/launcher/ee/allowedcmd"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const allowedCharacters = "0123456789"
+
+type Table struct {
+	slogger *slog.Logger
+	execCC  allowedcmd.AllowedCommand
```

Comment:
You don't need the `execCC` thing. It should only get used when we want to override commands in the test suite. Otherwise, just skip it